src/docker-compose.yml

```diff
@@ -76,6 +76,14 @@ services:
     command: ["/opt/spark/bin/spark-submit", "--packages",
       "org.apache.spark:spark-sql-kafka-0-10_2.12:3.3.2,com.microsoft.azure:spark-mssql-connector_2.12:1.3.0-BETA,com.microsoft.sqlserver:mssql-jdbc:12.2.0.jre8",
       "--conf", "spark.driver.extraJavaOptions=-Divy.cache.dir=/tmp -Divy.home=/tmp",
+      "--conf", "spark.ui.prometheus.enabled=true",
+      "--conf", "spark.executor.processTreeMetrics.enabled=true",
+      "--conf", "spark.metrics.conf.*.sink.prometheusServlet.class=org.apache.spark.metrics.sink.PrometheusServlet",
+      "--conf", "spark.metrics.conf.*.sink.prometheusServlet.path=/metrics/prometheus",
+      # "--conf", "spark.metrics.conf.master.sink.prometheusServlet.path=/metrics/master/prometheus",
```

Comment:
Even if I enable it, this endpoint is not working, and returns 404. Probably the Spark application is too simple and there is no master? But I left the configuration commented, just to have everything in one place, as the Spark monitoring documentation is quite scattered.
`/metrics/master/prometheus` endpoint should have the same data as `/metrics/master/json` (this one returns 404 too).